src/templates/apiPage.tsx

```diff
@@ -113,7 +113,11 @@ export default props => {
 
           {data.description && (
             <div className="pb-3 content-flush-bottom">
-              <p>{parseBackticks(data.description)}</p>
```

Comment:
is it possible for the description to contain other html tags? such as a `<script>` tag? if so we'll need to do something differently here to avoid xss

Reply:
@dcramer so far, we only use <a></a> and <code></code> equivalents in the OpenAPI schema.

Reply:
yeah but if something else exists will it just insert it? if so lets use dompurify here to restrict what html tags are possible

also could you not just render data.description as raw markdown somehow? e.g. how we did it before where the remark processor created a markdown node